src/ocamlorg_frontend/pages/tutorial.eml

```diff
@@ -1,5 +1,18 @@
 open Ocamlorg
 
+let to_kebab_case (s: string) = 
+  s
+  |> String.lowercase_ascii
+  |> String.to_seq
+  |> Seq.filter (fun c -> (c >= 'a' && c <= 'z') || (c >= '0' && c <= '9') || c = '-' || c = ' ')
+  |> String.of_seq
+  |> String.split_on_char ' '
+  |> String.concat "-"
```

Comment:
Is `to_kebab_case` doing the same thing as [`Utils.slugify`](../blob/main/src/ocamlorg_frontend/utils.ml)?

(Why wasnt't it named like you did in the first place, so nicer)

Reply:
Looks like it is. Will trying using that.

Reply:
Done